### ISSUE TITLE
Is there any training logs?

### ISSUE THREAD

**Issue by hyq12358** [yhenon/pytorch-retinanet#262]
I want to analyze the loss. 
Does any training log to download?
